### DISCUSSION TITLE
gm everyone! let's recover this project.

### DISCUSSION THREAD

**Discussion by KunalBagaria** [KunalBagaria/rustyvibes#38] [Announcements]
# Future updates and GUI

I never thought this project would get the traction from the open-source community that it ended up getting. It has been a long while since I built this project and was my first tool ever on rust. I'm excited to share that me & my friend would be working on an improved, revamped version of this CLI that would also be shipped in the form of a GUI app for macOS/Windows/Linux. Would love to hear your thoughts :)

reach out to me on twit: https://twitter.com/kb24x7

**Comment by redoxahmii** [comment]
Do provide the alternative to only use the cli as some would prefer not having a complete GUI app included as well.
One of the main reason i really liked this tool was i did not need a GUI app running all the time and the backwards compatibility of the audio packs available on mechvibes.
Hopefully the breaking changes and wayland support is implemented first and the rest can than be a major release.

**Comment by brushy501** [comment]
Great! The project is great, though i use Microsoft Windows, if it has a windows service to automatically run on startup will be great for me. But i know how much work has to go into the project. So keep it up and i hope the project keeps growing. 👍👍👍👍
